glotaran/project/result.py

```diff
@@ -310,6 +310,39 @@ def verify(self) -> bool:
 
         return True
 
+    def create_clp_guide_dataset(self, dataset: str, clp_label: str) -> xr.Dataset:
+        """Create dataset for clp guidance.
+
+        Parameters
+        ----------
+        dataset : str
```

Comment:
Renaming `dataset` -> `dataset_name` would make it more clear that a string is expected.
And it would allow more readable naming in the code
```diff
- result = self.data[dataset]
+ dataset = self.data[dataset_name]
```